playbooks/ansible.cfg

```diff
@@ -0,0 +1,18 @@
+[ssh_connection]
+pipelining=True
+ssh_args = -o ControlMaster=auto -o ControlPersist=30m -o ConnectionAttempts=100 -o UserKnownHostsFile=/dev/null
+[defaults]
+# https://github.com/ansible/ansible/issues/56930 (to ignore group names with - and .)
+force_valid_group_names = ignore
+
+host_key_checking=False
```

Comment:
Do we really want to have this disabled? It's annoying but it's there for a reason.

Reply:
I would like to disable it as it is annoying and kubespray them self have it disabled. So i thought it would be okay.

Reply:
Still not 100% convinced about this. I totally agree that it's annoying but I still think that this is a quite nice security feature.

Reply:
I think it is fine to disable

Reply:
I agree with you @Xartos  that it is a nice security feature. The most annoying thing is the first time running play books as the `add host to known_hosts` messes up the output. So that being said I will keep the config and set `host_key_checking=True`. So if someone wants to change it to false for a run it is easy to do so.